### ISSUE TITLE
Fix ID references in `who` attributes

### ISSUE THREAD

**Issue by cmil** [dracor-org/romdracor#2]
Instead of

https://github.com/dracor-org/romandracor/blob/816c278c535280579e9f005b2e2c50591dad6320/tei/seneca-medea.xml#L506

character IDs in `who` attributes need to be referenced like this
```xml
<sp who="#medea #nutrix">
```

**Comment by nevmenandr** [comment]
This is the problem only with Seneca tragedies. I think I fixed it.

**Comment by lehkost** [comment]
This example is a monologue by Medea (not a speech by the two characters mentioned). So I guess "Medea Nvtrix" should be wrapped as ```<stage>Medea Nvtrix</stage>``` and then ```<sp who="#medea">```.

Since this is unclear from the text and markup itself, I think we have to manually check passages like this.

**Comment by nevmenandr** [comment]
Yes, this strange way of encoding relates to all Seneca plays. I've fixed it.
